apt_install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -eux
+
+sudo apt-get purge ruby || true
```

Comment:
why?

Reply:
The GitHub hosted runners come with the `ruby` package installed by default, so trying to install the `ruby` package with this action will cause a conflict.